packages/core/schematics/collection.json

```
@@ -15,6 +15,14 @@
       "aliases": [
         "control-flow"
       ]
+    },
+    "standalone-routes-lazy-load-migration": {
+      "description": "Converts the entire application to lazy load standalone components",
+      "factory": "./ng-generate/standalone-routes/bundle",
```

I'd propose renaming the migration to something like this:
```suggestion
      "factory": "./ng-generate/route-lazy-loading/bundle",
```